src/routes/Missions.jsx

```diff
@@ -0,0 +1,88 @@
+import { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import Table from 'react-bootstrap/Table';
+import Button from 'react-bootstrap/Button';
+import Badge from 'react-bootstrap/Badge';
+import {
+  getMissionItems,
+  joinMission,
+  leaveMission,
+} from '../components/redux/missions/missionsSlice';
+import styles from '../styles/Missions.module.css';
+
+const Missions = () => {
+  const dispatch = useDispatch();
+  const { missions, isLoading, error } = useSelector((state) => state.missions);
+  useEffect(() => {
+    if (missions.length === 0) {
+      dispatch(getMissionItems());
+    }
+  }, [dispatch, missions.length]);
+  const handleJoinMission = (id) => {
+    dispatch(joinMission(id));
+  };
+
+  const handleLeaveMission = (id) => {
+    dispatch(leaveMission(id));
+  };
+
+  if (error) return <h2>Something went wrong</h2>;
+
+  return isLoading ? (
+    <h1>Loading...</h1>
+  ) : (
+    <section className={styles.wrapper}>
+      <hr />
+      <div className={styles.table}>
+        <Table striped bordered hover responsive>
+          <thead>
+            <tr>
+              <th>Mission</th>
+              <th>Description</th>
+              <th>Status</th>
+              <th>{'    '}</th>
+            </tr>
+          </thead>
+          <tbody>
+            {missions.map((mission) => (
+              <tr key={mission.mission_id}>
+                <td>{mission.mission_name}</td>
+                <td className={styles.desc}>{mission.description}</td>
+                <td>
+                  <div className={styles.center}>
+                    {mission.reserved ? (
+                      <Badge bg="success">Active Member</Badge>
+                    ) : (
+                      <Badge bg="secondary">Not A Member</Badge>
+                    )}
+                  </div>
+                </td>
+                <td>
+                  <div className={styles.centerBtn}>
+                    {mission.reserved ? (
+                      <Button
+                        variant="outline-danger"
+                        onClick={() => handleLeaveMission(mission.mission_id)}
+                      >
+                        Leave Mission
+                      </Button>
+                    ) : (
+                      <Button
+                        variant="outline-secondary"
+                        onClick={() => handleJoinMission(mission.mission_id)}
+                      >
+                        Join Mission
+                      </Button>
+                    )}
+                  </div>
+                </td>
+              </tr>
+            ))}
+          </tbody>
+        </Table>
+      </div>
```

Comment:
- [ ]  @Bifabig and @Mar12358 great work on your missions page, however, the color of your active member status icon and leave mission button don't match the design given check the comparison below:


Given Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/Bifabig/space-travelers-hub-react-redux/assets/61408860/e42faf1d-3dbd-48e5-8be2-6b772179cece) | ![image](https://github.com/Bifabig/space-travelers-hub-react-redux/assets/61408860/63076f4b-abf5-4e05-8c03-7da9c1c513d9)

Reply:
@danmainah the same goes for the leave mission button. Your screenshot has the cursor hovering over the button hence the color difference.